session/pingpong/consumer_balance_tracker.go

```diff
@@ -142,7 +144,18 @@ func (cbt *ConsumerBalanceTracker) handleUnlockEvent(id string) {
 	if err != nil {
 		log.Error().Err(err).Msg("Could not recover Grand Total Promised")
 	}
-	cbt.ForceBalanceUpdate(identity)
+
+	status, err := cbt.registry.GetRegistrationStatus(identity)
+	if err != nil {
+		log.Error().Err(err).Msg("Could not recover get registration status")
```

Comment:
Would be nice to somehow reuse same `handleRegistrationEvent(event registry.AppEventIdentityRegistration)` after force syncing this event.

Reply:
there might not be such events in case a node is restarted.